torch/onnx/_internal/exporters/dynamo_export.py

```diff
@@ -0,0 +1,25 @@
+import copy
+
+import torch._dynamo
+from torch.onnx._internal.exporter import ExportOutput
+from torch.onnx._internal.exporters.fx_base import FXGraphModuleExporter
+
+
+class DynamoExportExporter(FXGraphModuleExporter):
```

Comment:
Same as above (new stuff not discussed on v1 design proposal which deviates from the complete design)